about.html

```diff
@@ -13,42 +13,50 @@
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-    <nav class="desktop-nav">
+  <div class="social-media">
+    <ul class="links">
+      <li><i class="fa fa-facebook"></i></li>
+      <li><i class="fa fa-twitter"></i></li>
+      <li>Logout</li>
+      <li>My page</li>
+      <li>korean</li>
+    </ul>
+  </div>
+    <header class="desktop-nav">
         <div class="logo">
-         <a href="index.html"> <img src="image/logo5.png" alt="" /></a>
-        </div>
-  
-        <div class="nav-desktop-links">
-          <ul>
-            <li><a href="about.html">About</a></li>
-            <li><a href="#">Feature</a></li>
-            <li><a href="#">Register</a></li>
-          </ul>
+         <a href="index.html"> <img src="image/logo77.png" alt="" /></a>
```

Comment:
Great design @SAZZAD1Q2 ✋

- [ ] Just like the previous reviewer suggested, kindly add an `alt` attribute to your images. This is a good practice for semantic purposes.

Reply:
But I already added it. Why are you asking again that.